packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -421,8 +421,20 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
     CameraUpdate cameraUpdate, {
     required int mapId,
   }) {
-    return channel(mapId).invokeMethod<void>('camera#animate', <String, Object>{
+    return channel(mapId).invokeMethod<void>('camera#animate',
+        <String, dynamic>{'cameraUpdate': cameraUpdate.toJson()});
```

Comment:
Why was this changed to `dynamic`? We only use `dynamic` when we have an extremely compelling reason to do so as it bypasses a lot of static analysis.